### DISCUSSION TITLE
sd-dynamic-thresholding not work in hires.fix

### DISCUSSION THREAD

**Discussion by yamasoo** [vladmandic/sdnext#1264] [General]
As title

Version 
updated: 2023-06-01
hash: 561ebc61

sd-dynamic-thresholding
f02cacfc
Mon May22 2023 22:20

seems not work in hirex.fix

any one know how to fix it?

**Comment by vladmandic** [comment]
"seems not work in hirex.fix" is not really telling much - what is the problem?

**Comment by yamasoo** [comment]
1. Using CFG 20 ,dynamic-thresholding Mimic Scale Scheduler 7 helf-cosin up CFG Scale Scheduler 3.5 helf-cosin up  without hires.fix
it work well like before

https://imgur.com/1wRc5uc

2. after hires.fix  Upscale by 2  get the resault is just  like the resault by CFG=20 ,seems  dynamic-thresholding not working on hires.fix phase

https://imgur.com/LcrDKDF

**Reply by vladmandic** [reply]
which sampler are you using?

**Reply by yamasoo** [reply]
DPM++ 2M Karas Euler A...
Upscaler  use ESRGAN_4X Latent....etc

a lot of change  but all not work

**Comment by radry** [comment]
The "fix" is to not use highres fix but use img2img zu upscale the result instead. Then you don't need CFG tresholding

**Comment by yamasoo** [comment]
![image](https://github.com/vladmandic/automatic/assets/73381291/47e4082b-138e-4fc7-adf5-0c8d6cb97056)
The problem was solved by manually selecting NONE for this two option in the sampler settings and then saving .
But I'm not sure why
The version is the latest
da38d27f
